examples/app.py

```diff
@@ -85,7 +88,7 @@
     CELERY_EAGER_PROPAGATES_EXCEPTIONS=True,
     CELERY_RESULT_BACKEND="cache",
     DEBUG=True,
-    SEARCH_UI_SEARCH_API='invenio_search_ui.api',
+    SEARCH_UI_SEARCH_API='invenio_records_rest.recid_list',
```

Comment:
@drjova `invenio-search-ui` is by default registered on different Flask instance.

- `app` - "standard application" (`/login`, `/search`, ...)
- `api_app` - contains all REST-API endpoint with custom exception handling (`/api/...`)